Easydict/Feature/Service/CustomOpenAI/CustomOpenAIService.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  CustomOpenAIService.swift
+//  Easydict
+//
+//  Created by phlpsong on 2024/2/16.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Alamofire
+import CryptoKit
+import Defaults
+import Foundation
+
+@objc(EZCustomOpenAIService)
+class CustomOpenAIService: OpenAILikeService {
+    // MARK: Lifecycle
+
+    override init() {
+        super.init()
+        #if DEBUG
+        defaultAPIKey = "NnZp/jV9prt5empCOJIM8LmzHmFdTiVa4i+mURU8t+uGpT+nDt/JTdf14JglJLEwpXkkSw+uGgiE8n5skqDdjQ=="
+            .decryptAES()
+        #else
+        defaultAPIKey = "NnZp/jV9prt5empCOJIM8LmzHmFdTiVa4i+mURU8t+uGpT+nDt/JTdf14JglJLEwVm8Sup83uzJjMANeEvyPcw=="
+            .decryptAES()
+        #endif
+    }
+
+    // MARK: Public
+
+    override public func name() -> String {
+        NSLocalizedString("custom_openai", comment: "The name of Custom OpenAI Translate")
+    }
+
+    override public func link() -> String? {
+        "https://chat.openai.com"
+    }
+
+    // MARK: Internal
+
+    static let defaultModels = ["gpt-3.5-turbo-0125", "gpt-4-0125-preview"]
```

Comment:
No, we don't need to set default models here, just show placeholder enough.

Also, it looks like there's a bug here,  if you delete all the models, it gets set to the default models again.

<img width="856" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/3319561f-1f21-4d96-b007-4ddb9ff7d06c">

Reply:
The behavior is by design since I think it need to assign a default model here. 

I will update this part later.